internal/exporter/pulsarexporter/README.md

```diff
@@ -1,3 +1,12 @@
 # Pulsar Exporter
 
-Pulsar exporter exports metrics to Pulsar.
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [deprecated]          |
+| Supported pipeline types | metrics |
+
+The Pulsar exporter is used to export metrics to Apache Pulsar.
+
+The exporter is deprecated and will be replaced by the [OpenTelemetry Collector upstream exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/pulsarexporter).
+
+[deprecated]:https://github.com/open-telemetry/opentelemetry-collector#deprecated
```

Comment:
Should we show a warning log message as well?

Reply:
yes I'll add that.